homeassistant/components/mysensors.py

```diff
@@ -228,3 +223,101 @@ def node_update(update_type, node_id):
                 callback(self, node_id)
 
         return node_update
+
+
+class MySensorsDeviceEntity(object):
+    """Represent a MySensors entity."""
+
+    # pylint: disable=too-many-arguments,too-many-instance-attributes
+
+    def __init__(
+            self, gateway, node_id, child_id, name, value_type, child_type):
+        """
+        Setup class attributes on instantiation.
+
+        Args:
+        gateway (GatewayWrapper): Gateway object.
+        node_id (str): Id of node.
+        child_id (str): Id of child.
+        name (str): Entity name.
+        value_type (str): Value type of child. Value is entity state.
+        child_type (str): Child type of child.
+
+        Attributes:
+        gateway (GatewayWrapper): Gateway object.
+        node_id (str): Id of node.
+        child_id (str): Id of child.
+        _name (str): Entity name.
+        value_type (str): Value type of child. Value is entity state.
+        child_type (str): Child type of child.
+        battery_level (int): Node battery level.
+        _values (dict): Child values. Non state values set as state attributes.
+        mysensors (module): Mysensors main component module.
+        """
+        self.gateway = gateway
+        self.node_id = node_id
+        self.child_id = child_id
+        self._name = name
+        self.value_type = value_type
+        self.child_type = child_type
+        self.battery_level = 0
+        self._values = {}
+        self.mysensors = get_component('mysensors')
```

Comment:
Yeah, this is a leftover. I removed all the other calls to get_component.

Is there a clean way of using get_component in the global scope in the platforms, so that I could still reference the component when inheriting from ```mysensors.MySensorsDeviceEntity```, but not ignore a custom component?